pkg/http.ts

```diff
@@ -15,10 +15,21 @@ export type UpstashRequest = {
    * Request body will be serialized to json
    */
   body?: unknown;
+
+  upstashSyncToken?: string;
 };
 export type UpstashResponse<TResult> = { result?: TResult; error?: string };
 
 export interface Requester {
+  /**
+  * When this flag is not disabled, new commands of this client expects the previous write commands to be finalized before executing.
```

Comment:
When this flag is enabled, any subsequent commands issued by this client are guaranteed to observe the effects of all earlier writes submitted by the same client.